dev/preview/previewctl/pkg/preview/preview.go

```diff
@@ -55,18 +60,21 @@ func New(branch string, logger *logrus.Logger) (*Preview, error) {
 	}
 
 	return &Preview{
-		branch:     branch,
-		namespace:  fmt.Sprintf("preview-%s", GetName(branch)),
-		name:       GetName(branch),
-		kubeClient: harvesterConfig,
-		logger:     logEntry,
+		branch:          branch,
+		namespace:       fmt.Sprintf("preview-%s", GetName(branch)),
+		name:            GetName(branch),
+		kubeClient:      harvesterConfig,
+		logger:          logEntry,
+		vmiCreationTime: nil,
 	}, nil
 }
 
-func (p *Preview) InstallContext(watch bool, timeout time.Duration) error {
+func (p *Preview) InstallContext(wait bool, timeout time.Duration) error {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
+	p.logger.WithFields(logrus.Fields{"branch": p.branch, "timeout": timeout}).Infof("Installing context")
```

Comment:
```suggestion
	p.logger.WithFields(logrus.Fields{"timeout": timeout}).Infof("Installing context")
```

On line 55 we're already adding the branch to all logs from this logger, there is no need to add it here 🙂

Reply:
Thanks, I have accepted the change ✅